client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -343,6 +358,20 @@ class EmailProvidersComparison extends React.Component {
 				? newUsers( selectedDomainName )
 				: this.state.googleUsers;
 
+		const buttonLabel = this.isUpgrading( domain )
```

Comment:
Same sentiments about the domain argument being unneeded.

Reply:
I found some other places with the same issue 🤦‍♂️.